### DISCUSSION TITLE
Hi everyone, I am Isha.

### DISCUSSION THREAD

**Discussion by ishahah** [MDAnalysis/mdanalysis#4514] [Outreachy Discussions]
I might be a little late figuring out all the "dos" and "whats" of open source and MDAnalysis, because of being a newcomer. I am happy to finally connect and be learning from the community.

**Comment by hmacdope** [comment]
Welcome @ishahah! Have a read of our user guide, particularly the contributing section: https://userguide.mdanalysis.org/stable/contributing.html

**Comment by ishahah** [comment]
I've taken an interest in Project 1, mentioned under Outreachy Projects for MDAnalysis. I am about to fill the pre-proposal form to it. One thing I have not gotten to understand is that, do I still need to contribute?
